src/errors.rs

```diff
@@ -141,6 +141,12 @@ pub enum RustupError {
         target: TargetTriple,
         suggestion: Option<String>,
     },
+    #[error(
+        "unable to correctly link proxies\n\
+        this might be a bug in rustup, please open a new issue here:\n\
+        https://github.com/rust-lang/rustup/issues/new"
```

Comment:
```suggestion
        help: this might be a bug in rustup, please open a new issue here:\n\
        help: https://github.com/rust-lang/rustup/issues/new"
```